mne/viz/backends/renderer.py

```diff
@@ -158,7 +155,10 @@ def _get_3d_backend():
 
 @contextmanager
 def use_3d_backend(backend_name):
-    """Create a viz context.
+    """Create a 3d visualization context using the designated backend.
+
+    See :func:`mne.viz.set_3d_backend` for more details on the available
+    3d backends and their capabilities.
```

Comment:
can you also list the allowed backends below with a note that `matplotlib` backend is allowed only for one function?

Reply:
`matplotlib` is not a 3d backend.

Reply:
I don't want to bring confusion by talking about it since it's local to the function. I still think the function's doctring is the best place for that.

Reply:
But it's a public function, so document it, no? Otherwise, how is a user supposed to use it? Or do you mean the user shouldn't really use it but they can if they are desperate? :)

Reply:
Maybe I misunderstood. What should be added in the docstring of `plot_source_estimates`?

https://github.com/mne-tools/mne-python/blob/063ebc05de6931a28a348b0a9779f17801c68dfe/mne/viz/_3d.py#L1757-L1761

Reply:
oops :) I was talking about *this* function `use_3d_backends` down below on L165, something like:

```suggestion
    backend_name : 'mayavi' | 'pyvista' | 'notebook'
```

sorry that become more complicated than it needed to be.